addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -484,7 +484,7 @@
         * @private
         */
        _afterJoin: function () {
-            this.isMember = true;
```

Comment:
Why are we removing this line though? Is that necessary for the fix?

If not, I would keep it, even if not used or not relevant, to avoid undesired side-effects during the forward-port.

Reply:
@awa-odoo
It is not necessary for the fix.
However, the deleted line allows to set the `isMember` attribute to `true` for this slide only. The fix will set the `isMember` attribute to `true` for all slides including the current slide. Therefore, the line is no longer needed.

Reply:
That will not change the variable "this.isMember" of this specific widget to true anymore though.
So I still vote to keep it, there is no downside to keeping it and removing it could break things.